plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -321,6 +313,40 @@ open class HtmlRenderer(
         }
     }
 
+    private fun groupDivergentInstancesWithSourceSet(
```

Comment:
This made `buildDivergent` considerably longer, even though this particular function needed no context from it and shared nothing in common, so I don't see a reason why it has to be a nested function.

Extracted for the sake of readability of `buildDivergent`